pkg/utils/baremetal/bootstrap.go

```diff
@@ -92,6 +95,17 @@ func WaitForBaremetalBootstrapControlPlane(ctx context.Context, config *rest.Con
 				}
 			}
 
+			data, err := json.Marshal(masters)
+			if err != nil {
+				return true, err
+			}
+
+			mastersFilename := filepath.Join(dir, "masters.json")
```

Comment:
We should export the constant so we don't have to repeat the literal filename here.